.github/workflows/ci-windows-installed.yml

```diff
@@ -50,6 +63,7 @@ jobs:
         mv ${{ github.workspace }}\tmp\rav1e-windows-msvc-sdk\include\* ${{ github.workspace }}\vcpkg\installed\x64-windows-release\include
         mv ${{ github.workspace }}\tmp\rav1e-windows-msvc-sdk\lib\rav1e* ${{ github.workspace }}\vcpkg\installed\x64-windows-release\lib
         mv ${{ github.workspace }}\tmp\rav1e-windows-msvc-sdk\lib\pkgconfig\* ${{ github.workspace }}\vcpkg\installed\x64-windows-release\lib\pkgconfig
+    - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Can we replace egor-tensin/vs-shell with ilammy/msvc-dev-cmd? We should not need both of them.

Do you know why we can build libyuv using clang-cl in ci-windows.yml and ci-windows-artifacts.yml without ilammy/msvc-dev-cmd?

Reply:
Yes, so we want to call CMake with two different compilers. There are several solutions:
- set up two jobs, running conditionally on matrix.compiler. I did not do that because the CMake command is quite long
- have one job and set the compilers through `CC` and `CXX`. But I do not know what to put for `CC` for `msvc`. Setting it as empty creates an error: CMake believes there is no compiler. And env variables cannot be set conditionally, only jobs. The jobs you mention use `CC`.
- have one job and set the compilers through CMake. That is what I am doing here and it works.

Let me try to remove that line so that you can see the error.

Reply:
It now works without that line. Perfect ! Let's remove it.